common/lib/container-definitions/src/loader.ts

```diff
@@ -257,38 +248,38 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     /**
      * Provides the current connected state of the container
      */
-    readonly connectionState?: ConnectionState;
+    readonly connectionState: ConnectionState;
 
     /**
      * Boolean indicating whether the container is currently connected or not
      */
-    readonly connected?: boolean;
+    readonly connected: boolean;
 
     /**
      * Dictates whether or not the current container will automatically attempt to reconnect to the delta stream
      * after receiving a disconnect event
      * @param reconnect - Boolean indicating if reconnect should automatically occur
      * @alpha
```

Comment:
should alpha ever be non-optional? it seems it should stay optional, such that we can remove it, or change it without causing a break

Reply:
i figured we would probably be using alpha to mean it can break whenever, as in it's the easy tag to use for experimental APIs that ship as part of official releases.  then something like beta would confer more stability?
but otherwise agree with tony for this case.  if it's going to be promoted to required on a non-alpha interface, should be method still be considered alpha?

Reply:
the tags have no impact on the typescript build. changes to these will still lead to build breaks and are not semver compatible.

Reply:
right, more as a declaration of intent i mean

Reply:
for things that are truly alpha we probably want an independent interface that needs to be casted to to use, and still has all fields optional. i think this is a case where we want it to be alpha, but it's really not

Reply:
Fair point, I will update it accordingly

Reply:
Updated!